tests/foreman/cli/test_oscap.py

```diff
@@ -216,7 +256,7 @@ def test_negative_info_scap_content(self):
 
         :CaseImportance: Critical
         """
-        scap_id = gen_string('alphanumeric')
+        scap_id = gen_string('alpha')
```

Comment:
is scap id not a number?

Reply:
Negative test case, passing alphabet as ID. Will update the misleading variable name s/scap_id/invalid_scap_id.